presto-docs/src/main/sphinx/installation/spark.rst

```diff
@@ -0,0 +1,64 @@
+=========================
+Executing Presto on Spark
+=========================
+
+When there is a need to execute long running queries, Presto offers
+an option to scale the resource management using Spark, as it doesn'f offer
+one currently by itself. This way Presto can be executed as a stand alone 
+application on Spark.
+
+Steps
+-----
+
+Download the Presto Spark package tarball, :maven_download:`spark-package` 
+and the Presto Spark laucher, :maven_download:`spark-launcher`. Keep both the
+files at, say, *example* directory. We assume here a two node Spark cluster
```

Comment:
> with 2 cores each, thus giving us total four cores

If each spark workers has `2` cores then `task.concurrency`/`task.max-worker-threads`/`task.writer-count` and `spark.task.cpus`/`--executor-cores` must be set to `2`.

Reply:
wha's this "example" directory?  didn't see it get referred in command and other part of the doc?